tests/Neo.Network.RPC.Tests/Neo.Network.RPC.Tests.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\RpcClient\RpcClient.csproj" />
+    <ProjectReference Include="..\..\src\RpcServer\RpcServer.csproj" />
```

Comment:
Can you create a new UT project for RpcServer?

Reply:
The name should be `Neo.Plugins.RpcServer.Tests`.

Reply:
Yes, my latest commit has done this.

Reply:
Yours is `Neo.Plugin.RpcServer.Tests`.

Reply:
sorry for that, fixed already.